LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2018-2021 Iterative, Inc.
```

Comment:
I think it should only be one year but not sure which year. @shcheklein ?

Reply:
Hi, @jorgeorpinel! We've discussed this on slack and decided that we need to specify all years, not only one: https://iterativeai.slack.com/archives/C01SR9Q12LB/p1632989876075500